src/include/daos_api.h

```diff
@@ -166,6 +166,40 @@ daos_tx_restart(daos_handle_t th, daos_event_t *ev);
 int
 daos_tx_hdl2epoch(daos_handle_t th, daos_epoch_t *epoch);
 
+/**
+ * Initialize an iteratror anchor.
+ *
+ * \param[in]	anchor	Anchor to be initialized
+ * \param[in]	opts	(reserved) Initialization options
+ */
+static inline int
+daos_anchor_init(daos_anchor_t *anchor, unsigned int opts)
```

Comment:
You should update API_VERSION_MINOR in SConstruct so that a client can properly test for the presence of this API

Reply:
Indeed. Btw,  developers don't have to switch to these API straightaway but do have to eventually